packages/opentelemetry-cloud-trace-exporter/test/exporter.test.ts

```diff
@@ -58,9 +58,10 @@ describe('Google Cloud Trace Exporter', () => {
       });
     });
 
-    it('should construct exporter in GCE/GCP environment without args', async () => {
+    it('should construct exporter in GCE/GCP environment without args', async function () {
       // This variable is set by the test env and must be undefined to force
       // a metadata server request.
+      this.timeout(6000);
```

Comment:
This test specifically was having issues with the 2000ms timeout, not sure why.  This seems to address the issue.